### PR TITLE
EZP-26038: As an user, I want to be able to log in

### PR DESCRIPTION
**Pull request by clash82**
https://jira.ez.no/browse/EZP-26038

Review after merge of #37.

screen shot 2016-09-05 at 15 38 58

### CODE REVIEW

File: app/Resources/views/pagelayout.html.twig [hunk 1]
```diff
@@ -4,7 +4,10 @@
         {% if content is defined and title is not defined %}
             {% set title = ez_content_name(content) %}
         {% endif %}
-        <title>{{ title|default('Home')|trans }} - {{ 'eZ Platform Demo' }}</title>
+
+        {% block page_head %}
```
Comment (kamilmusial) [inline app/Resources/views/pagelayout.html.twig]: Isn't this part of #37 ?

**Comment by kamilmusial** [comment]
There are several parts of code for both #37 and #38 IMO that should be in one PR only. What do you think?

File: app/Resources/views/pagelayout.html.twig (hunk 1 above)
Comment (clash82) [inline app/Resources/views/pagelayout.html.twig]: Yes it is. It's build on top of #37 which must be merged first.

**Comment by kamilmusial** [comment]
+1

**Comment by damianz5** [comment]
+1

**Comment by SylvainGuittard** [comment]
+1

**Pull request merged**